meta/meta.go

```diff
@@ -21,19 +21,26 @@ import (
 	"github.com/pingcap/tidb/kv"
 )
 
-const (
+var (
 	// SchemaMetaPrefix is the prefix for database meta key prefix.
-	SchemaMetaPrefix = "mDB:"
+	SchemaMetaPrefix = string(MakeMetaKey("mDB:"))
+
 	// TableMetaPrefix is the prefix for table meta key prefix.
-	TableMetaPrefix = "mTable:"
-)
+	TableMetaPrefix = MakeMetaKey("mTable:")
 
-var (
 	nextGlobalIDPrefix = []byte("mNextGlobalID")
 	// SchemaMetaVersionKey is used as lock for changing schema
-	SchemaMetaVersionKey = []byte("mSchemaVersion")
+	SchemaMetaVersionKey = MakeMetaKey("mSchemaVersion")
+
+	// globalIDPrefix is used as key of global ID
+	globalIDKey = MakeMetaKey("mNextGlobalID")
 )
 
+// MakeMetaKey creates meta key
+func MakeMetaKey(key string) []byte {
+	return append([]byte{0x0}, key...)
```

Comment:
To separate from meta and data. Also we would never scan meta while scanning data.